backend/pkg/connect/create_connector_integration_test.go

```diff
@@ -0,0 +1,378 @@
+
+	seedBroker, err := getMappedHostPort(ctx, rpC, nat.Port(strconv.FormatInt(int64(exposedOutKafkaPort), 10)+"/tcp"))
+	require.NoError(err)
+
+	s.testSeedBroker = seedBroker
+
+	// HTTPBin container
+	httpC, err := runHTTPBin(ctx, connectTestNetwork)
+	require.NoError(err)
+
+	s.httpBinContainer = httpC
+
+	// Kafka Connect container
+	connectC, err := runConnect(connectTestNetwork, []string{"redpanda:" + strconv.FormatInt(int64(exposedPlainKafkaPort), 10)})
+	require.NoError(err)
+
+	s.connectContainer = connectC
+
+	connectPort, err := s.connectContainer.MappedPort(ctx, nat.Port("8083"))
+	require.NoError(err)
+
+	connectHost, err := s.connectContainer.Host(ctx)
+	require.NoError(err)
+
+	// Connect service
+	log, err := zap.NewProduction()
+	require.NoError(err)
+
+	connectSvs, err := NewService(config.Connect{
+		Enabled: true,
+		Clusters: []config.ConnectCluster{
+			{
+				Name: "redpanda_connect",
+				URL:  "http://" + connectHost + ":" + connectPort.Port(),
+			},
+		},
+	}, log)
+
+	require.NoError(err)
+
+	s.connectSvs = connectSvs
+}
+
+func (s *ConnectIntegrationTestSuite) TearDownSuite() {
+	t := s.T()
+	assert := require.New(t)
+
+	assert.NoError(s.redpandaContainer.Terminate(context.Background()))
+	assert.NoError(s.httpBinContainer.Terminate(context.Background()))
+	assert.NoError(s.connectContainer.Terminate(context.Background()))
+	assert.NoError(s.commonNetwork.Remove(context.Background()))
+}
+
+func (s *ConnectIntegrationTestSuite) TestCreateConnector() {
+	t := s.T()
+	require := require.New(t)
+	assert := assert.New(t)
+
+	ctx := context.Background()
+
+	t.Run("HTTP input happy path", func(t *testing.T) {
+		res, connectErr := s.connectSvs.CreateConnector(ctx, "redpanda_connect", connect.CreateConnectorRequest{
+			Name: "http_connect_input",
+			Config: map[string]interface{}{
+				"connector.class":                           "com.github.castorm.kafka.connect.http.HttpSourceConnector",
+				"header.converter":                          "org.apache.kafka.connect.storage.SimpleHeaderConverter",
+				"http.request.url":                          "http://httpbin:80/uuid",
+				"http.timer.catchup.interval.millis":        "10000",
+				"http.timer.interval.millis":                "10000",
+				"kafka.topic":                               "httpbin-input",
+				"key.converter":                             "org.apache.kafka.connect.json.JsonConverter",
+				"key.converter.schemas.enable":              "false",
+				"name":                                      "http_connect_input",
+				"topic.creation.default.partitions":         "1",
+				"topic.creation.default.replication.factor": "1",
+				"topic.creation.enable":                     "true",
+				"value.converter":                           "org.apache.kafka.connect.json.JsonConverter",
+				"value.converter.schemas.enable":            "false",
+			},
+		})
+
+		require.Empty(connectErr)
+		assert.NotNil(res)
+
+		timer := time.NewTimer(32 * time.Second)
+		<-timer.C
```

Comment:
What is this for? 32s seems very long.

Reply:
We run the HTTP input connect instance to do requests every `10s`, and we wait `32s` for it to do 3 requests, that then we verify below. I have changed the settings, so that the input does a request every `3s` and we wait `10s`. LMK if there are better settings, maybe `1.5s` and wait for `5s` ?

Reply:
I was wondering what the purpose of waiting was. Do you basically want to make sure the connector doesn't transition into an error state a few seconds after creating because it'll take a while until it gets to the running state? As far as Console is concerned I'd only check if the request was successful and possibly describe the created connector afterwards to see if it exist in the connect cluster no?

I don't want to fail if the connect cluster, conenctor plugin or the config itself has an issue because it wouldn't be addressable for us no?

Reply:
Oh ok... yes this is attempting full e2e type of test where we are verifying that the connector did its work properly...